init/redhat/py3/waagent.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
```

Comment:
how did you get the contents of this file?

Reply:
Context: We already have this file under `redhat` folder to handle py3 distro and also, we use `init `folder service file for py2. As part of this change, I have to modify these unit files to add slice info. Since `init `folder unit file being shared between all other distros, I moved original py3 file from `redhat `to `redhat/py3` path and copied common service to `redhat `folder. Hope this answered your question.

Reply:
yup, thanks